### ISSUE TITLE
Consider linking more prominently to GitHub

### ISSUE THREAD

**Issue by casey** [generative-music/generative.fm#213]
The link to GitHub on the help page is a bit hard to find, I think a lot of people would be interested in looking at the source code, so it might be good to put a link to GitHub on the front page.

**Comment by metalex9** [comment]
Agreed, I'll make sure it's more prominent in the next design. Thanks!
